src/wizard/Step1/components/RadioCards.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+<v-radio-group row @change="v=> $emit('change', v)">
+    <RadioCard class="radio-cards mr-2"
+      v-for="(item, index) in items"
+      :key="index"
+      :label="item.label"
+      :value="item.value"
+    />
+  </v-radio-group>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+import RadioCardItem from "./RadioCardItem";
+import RadioCard from "./RadioCard.vue";
+
+export default Vue.extend({
+  components: {
+    RadioCard,
+  },
+
+  props: {
+    items: {
+      type: Array,
+      default: () => Array<RadioCardItem>(),
+    },
+  }
+});
+</script>
+
+<style lang="scss">
```

Comment:
I figure this is just for dev purposes.  We always use this as a last case scenario.  If you need custom scss for this, create a file `_v_radio_button-cards.scss` in this folder `src\sass\components`.  You will need to add a reference to the file in this file `src\sass\atat.scss`